src/rpc/rawtransaction.cpp

```diff
@@ -1294,7 +1294,8 @@ UniValue blindpsbt(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
             "blindpsbt \"psbt\" ( ignoreblindfail )\n"
-            "\nUse the blinding data from the PSBT inputs to generate the blinding data for the PSBT outputs.\n"
+            "\nUse the blinding data from the PSBT inputs to generate the blinding data for the PSBT outputs.\n\n"
+            "TODO: Not expected to work on issuance/reissuance/peg transactions yet.\n"
```

Comment:
I'd rather not have a TODO in help text :) Just telling the user it isn't supported is enough

Reply:
Yeah fair. Fixing.